manifests/i/Iterative/DVC/3.57.0/Iterative.DVC.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Iterative.DVC
+PackageVersion: 3.57.0
+PackageLocale: en-US
+Publisher: Iterative
+PackageName: Data Version Control

```

Comment:
Can we rename this to `dvc`? Would that allow installing dvc via `winget install dvc`?